ganga/GangaDirac/Lib/Server/DiracCommands.py

```diff
@@ -405,3 +404,26 @@ def checkSEStatus(se, access = 'Write'):
     '''
     result = dirac.checkSEAccess(se, access)
     return result
+
+@diracCommand
+def getDDBtags( lfn ):
+    ''' returns the DDB and CONDDB tags of the last 
+	production step for a given LFN 
+    '''
+    from LHCbDIRAC.BookkeepingSystem.Client.BookkeepingClient import BookkeepingClient
+    import types
+    bk = BookkeepingClient()
+    prod = long(lfn.split('/')[5]) # not sure if this works in all cases 
+    res = bk.getProductionInformations( prod )
+    ddb = ''
```

Comment:
`ddb`  should be changed to `dddb` as it stands for Detector Description DataBase. This also applies to the changes in `GaudiExec.py`.